src/librustc_mir/transform/mod.rs

```diff
@@ -445,9 +448,11 @@ fn run_optimization_passes<'tcx>(
         // with async primitives.
```

Comment:
The comment here needs updating, it's not "inside the optimization block" any more.